boa/src/builtins/typed_array/mod.rs

```diff
@@ -38,6 +38,10 @@ macro_rules! typed_array {
             fn init(context: &mut Context) -> (&'static str, JsValue, Attribute) {
                 let _timer = BoaProfiler::global().start_event(Self::NAME, "init");
 
+                let typed_array_constructor = TypedArray::init(context);
+                let typed_array_constructor_proto =
+                    typed_array_constructor.get("prototype", context).unwrap();
```

Comment:
I would use a "expect" here to give some error message in case we mess up with the `TypedArray` initialization in a future Boa version.

Reply:
Yeah this is definitely not final. This also has the problem, that we call `TypedArray::init` for each typed array type, adding to the global init time. Also I'm not sure if this is spec compliant, as we build a `TypedArray` object for each typed array type, but in theory it should be the same object.